{"uid":"9cd9f61de64c5e28","name":"잘못된 ID,PW 입력 시 로그인이 실패하는지 확인","fullName":"src.tests.ui_tests.test_ui_login#test_login","historyId":"98e8f149c7227721782a965f730ba74a","time":{"start":1745925425003,"stop":1745925429004,"duration":4001},"status":"failed","statusMessage":"Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\n  (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55919be9a75a <unknown>\n#1 0x55919b94d4b0 <unknown>\n#2 0x55919b99e9b3 <unknown>\n#3 0x55919b99eba1 <unknown>\n#4 0x55919b9ed1f4 <unknown>\n#5 0x55919b9c45bd <unknown>\n#6 0x55919b9ea5e0 <unknown>\n#7 0x55919b9c4363 <unknown>\n#8 0x55919b990d63 <unknown>\n#9 0x55919b9919c1 <unknown>\n#10 0x55919be5fa6b <unknown>\n#11 0x55919be63951 <unknown>\n#12 0x55919be47b62 <unknown>\n#13 0x55919be644c4 <unknown>\n#14 0x55919be2c13f <unknown>\n#15 0x55919be886f8 <unknown>\n#16 0x55919be888d6 <unknown>\n#17 0x55919be995a6 <unknown>\n#18 0x7f98a4a9caa4 <unknown>\n#19 0x7f98a4b29c3c <unknown>","statusTrace":"src/tests/ui_tests/test_ui_login.py:22: in test_login\n    login_password_input = driver.find_element(By.ID, \"pw\")\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:770: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55919be9a75a <unknown>\nE   #1 0x55919b94d4b0 <unknown>\nE   #2 0x55919b99e9b3 <unknown>\nE   #3 0x55919b99eba1 <unknown>\nE   #4 0x55919b9ed1f4 <unknown>\nE   #5 0x55919b9c45bd <unknown>\nE   #6 0x55919b9ea5e0 <unknown>\nE   #7 0x55919b9c4363 <unknown>\nE   #8 0x55919b990d63 <unknown>\nE   #9 0x55919b9919c1 <unknown>\nE   #10 0x55919be5fa6b <unknown>\nE   #11 0x55919be63951 <unknown>\nE   #12 0x55919be47b62 <unknown>\nE   #13 0x55919be644c4 <unknown>\nE   #14 0x55919be2c13f <unknown>\nE   #15 0x55919be886f8 <unknown>\nE   #16 0x55919be888d6 <unknown>\nE   #17 0x55919be995a6 <unknown>\nE   #18 0x7f98a4a9caa4 <unknown>\nE   #19 0x7f98a4b29c3c <unknown>\n\nDuring handling of the above exception, another exception occurred:\nsrc/tests/ui_tests/test_ui_login.py:35: in test_login\n    pytest.fail(f\"❌ 로그인 실패 :{str(e)}\")\nE   Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55919be9a75a <unknown>\nE   #1 0x55919b94d4b0 <unknown>\nE   #2 0x55919b99e9b3 <unknown>\nE   #3 0x55919b99eba1 <unknown>\nE   #4 0x55919b9ed1f4 <unknown>\nE   #5 0x55919b9c45bd <unknown>\nE   #6 0x55919b9ea5e0 <unknown>\nE   #7 0x55919b9c4363 <unknown>\nE   #8 0x55919b990d63 <unknown>\nE   #9 0x55919b9919c1 <unknown>\nE   #10 0x55919be5fa6b <unknown>\nE   #11 0x55919be63951 <unknown>\nE   #12 0x55919be47b62 <unknown>\nE   #13 0x55919be644c4 <unknown>\nE   #14 0x55919be2c13f <unknown>\nE   #15 0x55919be886f8 <unknown>\nE   #16 0x55919be888d6 <unknown>\nE   #17 0x55919be995a6 <unknown>\nE   #18 0x7f98a4a9caa4 <unknown>\nE   #19 0x7f98a4b29c3c <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1745925424268,"stop":1745925425003,"duration":735},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\n  (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55919be9a75a <unknown>\n#1 0x55919b94d4b0 <unknown>\n#2 0x55919b99e9b3 <unknown>\n#3 0x55919b99eba1 <unknown>\n#4 0x55919b9ed1f4 <unknown>\n#5 0x55919b9c45bd <unknown>\n#6 0x55919b9ea5e0 <unknown>\n#7 0x55919b9c4363 <unknown>\n#8 0x55919b990d63 <unknown>\n#9 0x55919b9919c1 <unknown>\n#10 0x55919be5fa6b <unknown>\n#11 0x55919be63951 <unknown>\n#12 0x55919be47b62 <unknown>\n#13 0x55919be644c4 <unknown>\n#14 0x55919be2c13f <unknown>\n#15 0x55919be886f8 <unknown>\n#16 0x55919be888d6 <unknown>\n#17 0x55919be995a6 <unknown>\n#18 0x7f98a4a9caa4 <unknown>\n#19 0x7f98a4b29c3c <unknown>","statusTrace":"src/tests/ui_tests/test_ui_login.py:22: in test_login\n    login_password_input = driver.find_element(By.ID, \"pw\")\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:770: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55919be9a75a <unknown>\nE   #1 0x55919b94d4b0 <unknown>\nE   #2 0x55919b99e9b3 <unknown>\nE   #3 0x55919b99eba1 <unknown>\nE   #4 0x55919b9ed1f4 <unknown>\nE   #5 0x55919b9c45bd <unknown>\nE   #6 0x55919b9ea5e0 <unknown>\nE   #7 0x55919b9c4363 <unknown>\nE   #8 0x55919b990d63 <unknown>\nE   #9 0x55919b9919c1 <unknown>\nE   #10 0x55919be5fa6b <unknown>\nE   #11 0x55919be63951 <unknown>\nE   #12 0x55919be47b62 <unknown>\nE   #13 0x55919be644c4 <unknown>\nE   #14 0x55919be2c13f <unknown>\nE   #15 0x55919be886f8 <unknown>\nE   #16 0x55919be888d6 <unknown>\nE   #17 0x55919be995a6 <unknown>\nE   #18 0x7f98a4a9caa4 <unknown>\nE   #19 0x7f98a4b29c3c <unknown>\n\nDuring handling of the above exception, another exception occurred:\nsrc/tests/ui_tests/test_ui_login.py:35: in test_login\n    pytest.fail(f\"❌ 로그인 실패 :{str(e)}\")\nE   Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=135.0.7049.95); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55919be9a75a <unknown>\nE   #1 0x55919b94d4b0 <unknown>\nE   #2 0x55919b99e9b3 <unknown>\nE   #3 0x55919b99eba1 <unknown>\nE   #4 0x55919b9ed1f4 <unknown>\nE   #5 0x55919b9c45bd <unknown>\nE   #6 0x55919b9ea5e0 <unknown>\nE   #7 0x55919b9c4363 <unknown>\nE   #8 0x55919b990d63 <unknown>\nE   #9 0x55919b9919c1 <unknown>\nE   #10 0x55919be5fa6b <unknown>\nE   #11 0x55919be63951 <unknown>\nE   #12 0x55919be47b62 <unknown>\nE   #13 0x55919be644c4 <unknown>\nE   #14 0x55919be2c13f <unknown>\nE   #15 0x55919be886f8 <unknown>\nE   #16 0x55919be888d6 <unknown>\nE   #17 0x55919be995a6 <unknown>\nE   #18 0x7f98a4a9caa4 <unknown>\nE   #19 0x7f98a4b29c3c <unknown>","steps":[],"attachments":[{"uid":"3f5ce3de15ee7f09","name":"Failure Screenshot","source":"3f5ce3de15ee7f09.png","type":"image/png","size":168165},{"uid":"713eace7389cecb2","name":"stdout","source":"713eace7389cecb2.txt","type":"text/plain","size":81}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1745925429402,"stop":1745925429465,"duration":63},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"예외 케이스"},{"name":"story","value":"잘못된 ID,PW 입력"},{"name":"parentSuite","value":"src.tests.ui_tests"},{"name":"suite","value":"test_ui_login"},{"name":"host","value":"fv-az1693-462"},{"name":"thread","value":"2503-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"src.tests.ui_tests.test_ui_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"9cd9f61de64c5e28.json","parameterValues":[]}